src/common/src/idprovider/himmelblau.rs

```diff
@@ -49,7 +49,6 @@ use kanidm_hsm_crypto::{
 use regex::Regex;
 use reqwest;
 use reqwest::Url;
-use serde::{Deserialize, Serialize};
```

Comment:
I had to remove these lines and the `struct` below to get the workspace to build, otherwise I would not have touched this file.